### PR TITLE
allow copying advanced forms

### PR DESCRIPTION
**Pull request by snopoke**
@calellowitz 

cc @phillipm

### CODE REVIEW

**Comment by czue** [comment]
looks good on :+1:

File: corehq/apps/app_manager/models.py [hunk 1]
```diff
@@ -2168,6 +2168,9 @@ def uses_media(self):
     def uses_usercase(self):
         return False
 
+    def add_insert_form(self, from_module, form, index=None, with_source=False):
+        raise IncompatibleFormTypeException()
```
Comment (czue) [inline corehq/apps/app_manager/models.py]: just for my own education why was this necessary?

Comment (snopoke) [inline corehq/apps/app_manager/models.py]: Not striclty necessary but this method wasn't implemented by all the module subclasses so this is just a safety
